interface/java_binding/src/test/java/org/khronos/ktx/test/KtxTexture1Test.java

```diff
@@ -116,4 +117,21 @@ public void testCreate() {
 
         texture.destroy();
     }
+
+    @Test
+    public void testCreateFromBuffer() {
+        KtxTextureCreateInfo info = new KtxTextureCreateInfo();
+
+        info.setGlInternalformat(KtxInternalformat.GL_COMPRESSED_RGBA_ASTC_4x4_KHR);
+        info.setBaseWidth(10);
+        info.setBaseHeight(10);
+
+        KtxTexture1 texture = KtxTexture1.create(info, KtxCreateStorage.ALLOC);
+        assertNotNull(texture);
+
+        ByteBuffer imageData = ByteBuffer.allocateDirect(10 * 10);
```

Comment:
I think you should put some known data in the ByteBuffer and after `setImageFromBuffer`, check that the texture has the expected data.